content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+
+    * create-appliance.sh (ssh keys, ldap setup)
+    * setup-appliance.sh \[WKSHP-NAME] (Student setup, Appliance Setup, Workshop setup) calls:
+
+      * setup-globalappliance.sh (global / generic setup)
+      * setup-\[WKSHP-NAME].sh (Prepare appliance with workshop's reqs, Docker image for instance)
+    * create-\[WKSHP-NAME].sh (called at deployement time to instantiate the necessary appliance(s) requiered by the workshop 
+    * reset-appliance (reset ssh keys and students credentials on appliance
+    * cleanup-\[WKSHP-NAME].sh (takes care of cleanup some workshop's specifics)
+    * reset-\[WKSHP-NAME].sh (reset of the workshop's appliance, docker compose down of a container for instance)
+* A set of variables to be leveraged by the notebooks. These variables are to be set in yml format. They will be parsed at deployment times to set student ids, appliance IP addresses, and other relevant parameters like ports, simulated hardware information for instance.
+
+Whenever all the scripts are functional and that the necessary actions have been performed both on backend and frontend servers, some functional tests can conducted using cli and later webui as described earlier for the simple workshop example.
+
+* From the cli on the jupyterhub server: 
+
+  * one can leverage the wod-test-action.sh script to test a workshop lifecycle action from deployment (CREATE) to cleanup, rest or purge.
+
+    ```
+    dev@dev3:~$ wod-test-action.sh
+    Syntax: wod-test-action.sh <CREATE|CLEANUP|RESET|PURGE|PDF|WORD> WKSHOP [MIN[,MAX]
+    ACTION is mandatory
+    ```
+  * The available trace under ~/.mail/from will detail the different steps of the action and allow you to troubelshoot any issue.
+
+A﻿ set of notebooks that will be used by the student to follow instructions cells in markdown and run code cells leveraging the relevant kernel. If you are not familiar with Jupyter notebooks, a simple [101 workshop](https://developer.hpe.com/hackshack/workshop/25) is available in our Workshops-on-Demand 's catalog.
+
+A workshop should come up with at least :
+
+* 0-ReadMeFirst.ipynb 
+* 1-WKSHP-LAB1.ipynb
+* 2-WKSHP-LAB2.ipynb
+* 3-WKSHP-Conclusion.ipynb
+* LICENCE.MD
+* A pictures folder (if any screenshot is required in lab instructions)
+* A README.md (0-ReadMeFirst.ipynb in md format)
+* wod.yml ( for database injection)
+
+When all tests validated the workshop, it can follow the move to prod cycle.
+
+
+
+You should now have a better understanding of the necessary tasks associated to the creation of a workshop. As you can see, it requires steps on the various sides of the infrastructure. I will need to address one last point in this blog series related to the Worshops-on-Demand project, and this is the frontend part. I will explain how to deploy and mange it.
```

Comment:
Change the last sentence to read "**In my next post,** I will explain how to deploy and **manage** it."